uPortal-webapp/src/test/java/org/apereo/portal/io/xml/IdentityImportExportTestUtilities.java

```diff
@@ -81,6 +83,13 @@ public Object doInTransaction(TransactionStatus status) {
 
         // Make sure the data was exported
         assertNotNull("Exported data was null", dataExport);
+        if (resource.getFilename().equals("test_4-0.stylesheet-descriptor.xml")) {
```

Comment:
Why was this filename check necessary? It looks weird to have a specific filename (maybe used in some tests) to be in a utility class. I'm not sure if adding a boolean argument like `sort` would be better, but since you already have accepted PR in this repo, you can ask for the developer's opinion on this. Feel free to open an issue or real PR.